### PR TITLE
Add podcast

### PR DESCRIPTION
**Pull request by UniversalTraveller**

### CODE REVIEW

File: src/pages/add-podcast.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import AddPodcast from '../components/AddPodcast';
+import DisplayPodcasts from '../components/DisplayPodcasts';
+import Layout from '../components/Layout';
+import useStore from '../hooks/useStore';
+
+export default function AddPodcastPage() {
+	const podcasts = useStore(state => state.podcasts);
+	return (
+		<>
+			<AddPodcast></AddPodcast>
```
Comment (ahohnsen) [inline src/pages/add-podcast.js:12]:
```suggestion
			<AddPodcast />
```

File: src/utils/renderDate/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default function renderDate(dateString) {
+	const date = new Date(dateString);
+	const [month, day, year] = [date.getMonth(), date.getDate(), date.getFullYear()];
+	const formattedDate = `${day}.${month + 1}.${year}`;
+	return formattedDate;
```
Comment (ahohnsen) [inline src/utils/renderDate/index.js:5]: To simply display the date in a specific format, you could also use the method `.toLocaleDateString()` with the option `'de-De'` - check it out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Hey Andreas,

Well done 🚀 I added some comments 🙂

File: src/components/AddPodcast/index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+				<AddPodcastForm onSubmit={handleSubmit}>
+					<label htmlFor="feedUrl">
+						Please enter the feed URL of a podcast you want to add:
+					</label>
+					<input
+						type="text"
+						name="feedUrl"
+						id="feedUrl"
+						placeholder="https//podcast.com/feed.rss"
+					>
```
Comment (UniversalTraveller) [inline src/components/AddPodcast/index.js:60]: Self closing input tag still needs closing.
```suggestion
					/>
```

File: src/components/MainContentArea/styled.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const MainContentArea = styled.div`
+	margin: 2rem;
+`;
```
[the remark this answers is not in the document]
Comment (UniversalTraveller) [reply src/components/MainContentArea/styled.js:5]: That's how it was originally, but I want my notifications without margin. That's why I introduced this MainContentArea.

File: src/utils/renderDate/index.js (hunk 2 above)
Comment (UniversalTraveller) [reply src/utils/renderDate/index.js:5]: Awesome, thank you! I actually tried .toLocaleString() but thought it's useless outside of America. I missed the "de-DE" option.

File: src/components/AddPodcast/index.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+export default function AddPodcast() {
+	const router = useRouter();
+	const [message, setMessage] = useState('form');
+	const [addedPodcastTitle, setAddedPodcastTitle] = useState('');
+	const {addPodcast} = useStore();
+
+	function handleCancel() {
+		router.push('/');
+	}
```
Comment (ahohnsen) [inline src/components/AddPodcast/index.js:17]: You could also write this directly in the `onClick` of the button if you like :)

File: src/components/AddPodcast/index.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+		}
+
+		addPodcast(feedChannel); //add podcast to store
+		setAddedPodcastTitle(feedChannel.title); //set added podcast title for notification
+		setMessage('success');
+	}
+
+	function handleConfirmation() {
+		router.push('/');
+	}
```
Comment (ahohnsen) [inline src/components/AddPodcast/index.js:44]: same here

**Review by ahohnsen — APPROVED** [review]
Well done 👍🏼 I have a little suggestion, have a look 🙂

File: src/components/AddPodcast/index.js (hunk 5 above)
Comment (UniversalTraveller) [reply src/components/AddPodcast/index.js]: Thanks! That saves some lines of code.

**Pull request merged**